app/models/admin_user.rb

```diff
@@ -0,0 +1,18 @@
+class AdminUser < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, 
```

Comment:
Duplication Removed waqas bhai. It was my error, I accidentally hit crtl + v several times on that page. sorry